test/e2e/fixture/fixture.go

```diff
@@ -811,6 +811,8 @@ func RestartRepoServer() {
 		}
 		FailOnErr(Run("", "kubectl", "rollout", "restart", "deployment", workload))
 		FailOnErr(Run("", "kubectl", "rollout", "status", "deployment", workload))
```

Comment:
The `rollout status` should already wait until a workload is available. I think we should really avoid using introducing new`Sleep` calls with a fixed time length.

Why is this only needed on s390x?

Reply:
See `http://pastebin.test.redhat.com/1094074` for failed cases log,  on s390x  the `rollout status` does  not wait enough longer,  and causes the follow calling to argocd repo server failed